### PR TITLE
refactor: change the normalizer name from capatial to normal letter

### PR DESCRIPTION
**Pull request by kwangneuraco**

### CODE REVIEW

File: neuracore/ml/algorithms/cnnmlp/cnnmlp.py [hunk 1]
```diff
@@ -299,11 +299,11 @@ def __init__(
             self.proprio_dims[data_type] = (current_dim, current_dim + dim)
             current_dim += dim
 
-        self.action_normalizer = ACTION_NORMALIZER(
+        self.action_normalizer = action_normalizer(
             name="actions", statistics=output_stats
         )
         self.proprio_normalizer = (
-            PROPRIO_NORMALIZER(name="proprioception", statistics=input_stats)
+            proprio_normalizer(name="proprioception", statistics=input_stats)
```
Comment (aditya-neuraco) [inline neuracore/ml/algorithms/cnnmlp/cnnmlp.py:306]: Why not just call the normaliser class directly? Since we are assigning to the class attributes for action and proprio normalizer, it's pretty clear what we are doing.

File: neuracore/ml/algorithms/diffusion_policy/diffusion_policy.py [hunk 2]
```diff
@@ -212,11 +212,11 @@ def __init__(
         # Only create proprio_normalizer if there are proprioception stats
         # This allows the algorithm to work without proprioception (visual-only)
         self.proprio_normalizer = (
-            PROPRIO_NORMALIZER(name="proprioception", statistics=proprio_stats)
+            proprio_normalizer(name="proprioception", statistics=proprio_stats)
             if proprio_stats
             else None
         )
-        self.action_normalizer = ACTION_NORMALIZER(
+        self.action_normalizer = action_normalizer(
```
Comment (aditya-neuraco) [inline neuracore/ml/algorithms/diffusion_policy/diffusion_policy.py:219]: Same as above.

File: neuracore/ml/algorithms/groot/groot.py [hunk 3]
```diff
@@ -356,10 +356,10 @@ def __init__(
             f"max_action_dim={max_action_dim}"
         )
 
-        self.proprio_normalizer = PROPRIO_NORMALIZER(
+        self.proprio_normalizer = proprio_normalizer(
             name="proprioception", statistics=proprio_stats
         )
-        self.action_normalizer = ACTION_NORMALIZER(
+        self.action_normalizer = action_normalizer(
```
Comment (aditya-neuraco) [inline neuracore/ml/algorithms/groot/groot.py:362]: Same as above.

File: neuracore/ml/algorithms/groot/image_processor.py
Comment (aditya-neuraco) [inline neuracore/ml/algorithms/groot/image_processor.py:1]: This seems to be the change from another PR, please fix.

File: neuracore/ml/algorithms/groot/modules.py
Comment (aditya-neuraco) [inline neuracore/ml/algorithms/groot/modules.py:1]: This seems to be the change from another PR, please fix.

File: neuracore/ml/algorithms/groot/utils.py
Comment (aditya-neuraco) [inline neuracore/ml/algorithms/groot/utils.py:1]: This seems to be the change from another PR, please fix.

File: neuracore/ml/algorithms/pi0/pi0.py [hunk 4]
```diff
@@ -236,11 +236,11 @@ def __init__(
         # Only create proprio_normalizer if there are proprioception stats
         # This allows the algorithm to work without proprioception (visual-only)
         self.proprio_normalizer = (
-            PROPRIO_NORMALIZER(name="proprioception", statistics=proprio_stats)
+            proprio_normalizer(name="proprioception", statistics=proprio_stats)
             if proprio_stats
             else None
         )
-        self.action_normalizer = ACTION_NORMALIZER(
+        self.action_normalizer = action_normalizer(
```
Comment (aditya-neuraco) [inline neuracore/ml/algorithms/pi0/pi0.py:243]: Same comments as above.

File: neuracore/ml/algorithms/pi05/pi05.py [hunk 5]
```diff
@@ -264,11 +264,11 @@ def __init__(
...
         self.proprio_normalizer = (
-            PROPRIO_NORMALIZER(name="proprioception", statistics=proprio_stats)
+            proprio_normalizer(name="proprioception", statistics=proprio_stats)
             if proprio_stats
             else None
         )
-        self.action_normalizer = ACTION_NORMALIZER(
+        self.action_normalizer = action_normalizer(
             name="actions", statistics=output_stats
         )
```
Comment (aditya-neuraco) [inline neuracore/ml/algorithms/pi05/pi05.py:273]: Same comments as above.

**Review by aditya-neuraco — CHANGES_REQUESTED** [review]

File: neuracore/ml/algorithms/cnnmlp/cnnmlp.py (hunk 1 above)
Comment (kwangneuraco) [reply neuracore/ml/algorithms/cnnmlp/cnnmlp.py:306]: Unfortunately it is not that straightforward, we give user the freedom to change different normalizer

File: neuracore/ml/algorithms/groot/image_processor.py
Comment (kwangneuraco) [reply neuracore/ml/algorithms/groot/image_processor.py:1]: Yes already ask for your review

Comment (aditya-neuraco) [inline neuracore/ml/algorithms/groot/image_processor.py:1]: Please remove from this PR. It's part of https://github.com/NeuracoreAI/neuracore/pull/591

**Review by aditya-neuraco — APPROVED** [review]
LGTM! w/ comments

- Please separate the GROOT changes from this PR before merging. They are also part of https://github.com/NeuracoreAI/neuracore/pull/591
- Add a TO-DO note to have a look at the normalizer code and config again.

File: neuracore/ml/algorithms/groot/groot.py [hunk 6]
```diff
@@ -65,14 +65,14 @@
...
-    _load_pretrained_state_dict,
 )
+from .utils import _load_pretrained_state_dict
 
 logger = logging.getLogger(__name__)
 
 # Normalizer types for proprioception and actions
-PROPRIO_NORMALIZER = MeanStdNormalizer
-ACTION_NORMALIZER = MeanStdNormalizer
+proprio_normalizer = MeanStdNormalizer
```
Comment (mark-neuracore) [inline neuracore/ml/algorithms/groot/groot.py:74]: Question: why do we need to define this globally as an alias to `MeanStdNormalizer` like this and not just use `MeanStdNormalizer` down there?

**Comment by mark-neuracore** [comment]
github is glitching with comments in files. i think this pr has changes from another pr mixed up.

**Comment by mark-neuracore** [comment]
You can rebase this branch on top of the other pr and stack your prs and probably that would  solve the issue :)

File: neuracore/ml/algorithms/groot/groot.py (hunk 6 above)
Comment (kwangneuraco) [reply neuracore/ml/algorithms/groot/groot.py:74]: You can change to a different normalizer and it is easier for the user to change when put on top and in the future this will go to pre-processing
